src/webui/www/public/scripts/client.js

```diff
@@ -161,11 +161,20 @@ window.addEvent('load', function () {
         $('mochaToolbar').addClass('invisible');
     }
 
+    // Show Status Bar is enabled by default
+    var showStatusBar = true;
+    if (localStorage.getItem('show_status_bar') !== null)
+        showStatusBar = localStorage.getItem('show_status_bar') == "true";
```

Comment:
`show_status_bar`: you should use camelcase naming and please don't name it `showStatusBar` (already used).

Reply:
The existing files really need to be cleaned up. Following the current conventions in the file shouldn't lead to conflicts like these.

Reply:
It appears that all current localStorage properties use snake_case:

```
properties_height_rel
filters_width
selected_category
selected_filter
show_top_toolbar
speed_in_browser_title_bar
selected_tab
column_*
columns_order_*
sorted_column_*
reverse_sort_*
```